openownership/data-standard
# ChangeLog edits for clarity, fixes 418

# Overview

- What does this pull request do?

An edit of the changelog.rst page

- How can a reviewer test or examine your changes?

Additions to this file have been made at the point of merging related PRs into master. This has been great for ensuring changes are documented, but this final edit is needed to make sure that the language and markup is consistent and that changes are logged with a useful level of detail.

A reviewer should compare the old with the new text and make sure that no important, material information has been removed (which is not available elsewhere, e.g. in commit messages).

- Who is best placed to review it?

@siwhitehouse for the overall edit, @rhiaro for the 'Alterations to schema structure and logic' section.

(Closes/Relates to) issue: #418

## Translations

- n/a.... I've notified the translation coordinator of any new strings that will need
      translating. See: https://openownership.github.io/bods-dev-handbook/translations.html

## Documentation & Release

      https://openownership.github.io/bods-dev-handbook/standard_releases.html      
- n/a... I've updated [reference.rst](https://standard.openownership.org/en/latest/schema/reference.html) to reflect any changes to schema structure or naming.
- n/a... I've added or edited any other related documentation. See: https://github.com/openownership/bods-dev-handbook/blob/master/style_guide.md
